assets/javascripts/discourse/services/chat.js

```diff
@@ -470,15 +471,42 @@ export default Service.extend({
     });
   },
 
+  unfollowDirectMessageChannel(channel) {
```

Comment:
Related to above, then does this need `async`?

Reply:
yes I can do it :+1:

Reply:
It's turtles all the way down..........